README.md

```diff
@@ -64,6 +64,7 @@ Procuramos ordenar os conteúdos e criar uma navegação amigável, visando faci
 
 - [201 Front-End](https://www.youtube.com/channel/UCZXLQtYuKWpPq2Oaf5zpzhg) - Canal com conteúdos relacionados ao desenvolvimento Front-End. Interfaces amigáveis, programação de animações, estética de interfaces, plugins e bibliotecas, dentre outros assuntos. _Tags: `front-end`,`html`,`css`,`ui design`,`web design`_
 - [Algaworks](https://www.youtube.com/user/algaworks/playlists?view=1&flow=grid) - Canal da empresa de cursos online Algaworks, onde disponibilizam variados vídeos desde material de apoio sobre diversas tecnologias da atualidade até dicas para quem atua ou pretende atuar na área de desenvolvimento. _Tags: `html5`, `css`, `angularjs`, `angular`, `angular material`_
+- [Canal SW9 - FullStack](https://www.youtube.com/+sw9brl) - O Canal SW9 é mantido por Paulo Eduardo de Camargo. Engenheiro da Computação com mais de 15 anos de experiência em desenvolvimento web. A proposta do canal é apoiar a comunidade de desenvolvedores web e híbrido mobile compartilhando dicas, tutorias e aulas relacionadas com linguagens Back e Front, Análise de Ferramentas/Serviços Web, API, SEO e Trilhas de Conhecimento. Todos os códigos fontes estão disponíveis de graça em http://blog.sw9.com.br/acesso-area-de-downloads . _Tags: `Javascript`, `BootStrap UI`, `HTML5`,`CSS3`, `AngularJS`, `Angular 2+`, `NodeJS`, `PHP`, `API`, `TDD`, `Firebase`, `Gulp` _
```

Comment:
As tags não estão formatadas em itálico, eu acho que é pq tem um espaço antes do `_`
Sua descrição esta assim:

>  - [Canal SW9 - FullStack](https://www.youtube.com/+sw9brl) - O Canal SW9 é mantido por Paulo Eduardo de Camargo. Engenheiro da Computação com mais de 15 anos de experiência em desenvolvimento web. A proposta do canal é apoiar a comunidade de desenvolvedores web e híbrido mobile compartilhando dicas, tutorias e aulas relacionadas com linguagens Back e Front, Análise de Ferramentas/Serviços Web, API, SEO e Trilhas de Conhecimento. Todos os códigos fontes estão disponíveis de graça em http://blog.sw9.com.br/acesso-area-de-downloads . _Tags: `Javascript`, `BootStrap UI`, `HTML5`,`CSS3`, `AngularJS`, `Angular 2+`, `NodeJS`, `PHP`, `API`, `TDD`, `Firebase`, `Gulp` _

O certo seria assim:

>  - [Canal SW9 - FullStack](https://www.youtube.com/+sw9brl) - O Canal SW9 é mantido por Paulo Eduardo de Camargo. Engenheiro da Computação com mais de 15 anos de experiência em desenvolvimento web. A proposta do canal é apoiar a comunidade de desenvolvedores web e híbrido mobile compartilhando dicas, tutorias e aulas relacionadas com linguagens Back e Front, Análise de Ferramentas/Serviços Web, API, SEO e Trilhas de Conhecimento. Todos os códigos fontes estão disponíveis de graça em http://blog.sw9.com.br/acesso-area-de-downloads . _Tags: `Javascript`, `BootStrap UI`, `HTML5`,`CSS3`, `AngularJS`, `Angular 2+`, `NodeJS`, `PHP`, `API`, `TDD`, `Firebase`, `Gulp`_

Percebeu a diferença ? As tags estão em itálico, é simples resolver.. só remove o espaço em branco antes do último `_`

Reply:
Corrigido